app/models/stellar_base/deposit.rb

```diff
@@ -0,0 +1,7 @@
+module StellarBase
```

Comment:
This is a model that's in our app. It doesn't feel like it should be in the `StellarBase` namespace. Maybe I'm missing something...

Reply:
It makes sense if it is supplied and the users of the gem just conform to the fields available and get a bunch of free stuff. Omakase!

Reply:
It's still not clear to me, so should they just pass an `ActiveRecord` model and then we just `.send`? Or did I misunderstand the `FindDeposit` action in the original `Trigger` spec?

Reply:
What I had in mind was: App that uses SB has its own `Deposit` model so that it can ensure that when it calls `StellarBase::Deposits::Trigger`, it can be sure that it is unique.

Reply:
Though if `Deposit` is supplied by this engine, then the app that uses this engine can just call trigger which creates a deposit model and ensures that it only processes it once